module/datamanager.py

```diff
@@ -1000,21 +1024,42 @@ def get_overall_it_state(self, user):
         return hosts_state, services_state
 
     def get_important_elements(self, user, type='all', sorter=worse_first):
-        return self.search_hosts_and_services('bi:>2 ack:false type:%s' % type, user=user, sorter=sorter)
+        # BI is greater than the minimum business impact
+        # todo: @maethor: why searching with ack:false ? an element is important whether it is ack or not...
+        search = "bi:>%d " % self.min_business_impact + 'ack:false type:%s' % type
+        return self.search_hosts_and_services(search, user=user, sorter=sorter)
 
-    def get_impacts(self, user, search='is:impact bi:>=0 type:all', sorter=worse_first):
+    def get_impacts(self, user, search='is:impact type:all', sorter=worse_first):
         if "is:impact" not in search:
             search = "is:impact " + search
+        if "bi:" not in search:
+            search = "bi:>=%d " % self.min_business_impact + search
         return self.search_hosts_and_services(search, user=user, get_impacts=True, sorter=sorter)
 
-    def get_problems(self, user, search='isnot:UP isnot:OK isnot:PENDING bi:>=0 type:all',
-                     get_acknowledged=False, get_downtimed=False, sorter=worse_first):
+    def get_problems(self, user, search='isnot:UP isnot:OK isnot:PENDING isnot:UNKNOWN isnot:ACK isnot:DOWNTIME '
+                                        'is:HARD type:all', get_acknowledged=False, get_downtimed=False,
+                     sorter=worse_first):
```

Comment:
I disagree. UNKNOWN IS a problem. You can get an UNKNOWN when a check plugin doesn't work, if you filter on this you will never see that the plugin is not working (and the service may not be OK). Worse, if you don't configure thresholds properly, you can get an UNKNOWN on `check_http` when the site timeout. What does it mean ? Is it not a problem ?

I cannot merge that. Please don't change this query.

Reply:
Please do not remove the bi filter. As discussed by email, you need to differentiate "min_business_impact" for the top badges and for the default problems query.

In my case, I want to keep a look on the "low importance" problems, like pending upgrades or problems on staging environments. But I don't want this problems to be counted in the top badges, because they do not need immediate attention or fixing.

Reply:
It was a suggestion.

1. for UNKNOWN, it is a matter of view ... I can accept your analysis. I proposed to exclude UNKNOWN because Nagios does ... but I agree with you. I will revert this

1. for the BI filter I also agree. I propose another approachh ... each contact in the configuration has a `min_business_impact`  in its attributes and we should use this as a default filte for the problems view. The default value is 0 to view all the problems but if you set 2 for the user of the webui you will have a de-facto filter. It would make sense ...

Reply:
About point 2. it makes sense. But I think you still need to have two variables, one for problems view and one for the badge.

Reply:
Hmmm... I will also check if the datamanger considers the current user `min_bi`. And then we will have this order in the filtering:
 - user `min_bi` filtering first
 - main bi filtering for the whole UI (including problems view)
 - badge bi filtering for the badges